plotnine/tests/test_theme.py

```diff
@@ -143,6 +143,11 @@ def test_theme_minimal(self):
 
         assert p + _theme == 'theme_minimal'
 
+    def test_theme_tufte(self):
+        p = self.g + labs(title='Theme Tufte') + theme_tufte()
+
+        assert p + _theme == 'theme_tufte'
```

Comment:
You need to include a baseline image, as is the case for [all other themes](https://github.com/has2k1/plotnine/tree/f71d043c367927929fe0736ef6bc405767ff0e8d/plotnine/tests/baseline_images/test_theme).